apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -45,7 +46,9 @@ const SongSelector = Radium(class extends React.Component {
   };
 
   changeSong = (event) => {
-    this.props.setSong(event.target.value);
+    const song = event.target.value;
+    this.props.setSong(song);
+    project.saveSelectedSong(song);
```

Comment:
Are we still supporting this in Game Lab, or has it all moved over to the Dance app now?  I'm wondering if we need to update this at all, and/or should tear it out. @joshlory do you know?

Reply:
I don't think this is a priority.